src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -85,13 +98,14 @@ public String visitPost( AstBinaryOperation node, String lhs, String rhs ){
 				throw new RuntimeException( "not yet implemented" );
 			}
 			case COMPARE_UNEQUAL:{
-				throw new RuntimeException( "not yet implemented" );
+				return String.format( "%s%sif_icmpeq #0_eq%niconst_1%ngoto #0_end%n#0_eq:%niconst_0%n#0_end:%n", lhs, rhs);
```

Comment:
It would be better to have a counter for the labels to ensure uniqueness. However, the test is not sensitive enough for this. In this respect, we accept this for the time being.